### DISCUSSION TITLE
upcoming website?

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#13398] [Q&A] [answered]
This is mentioned here https://github.com/vectordotdev/vector/pull/13391#issuecomment-1171293009

Is this tracked somewhere in public? I look forward to it because the current one is a pain to navigate... it's slow, and has not-great search experience.

**Comment by jszwedko** [comment, marked as answer]
Hi @tshepang !

Thanks for the feedback (and all of your proofreading PRs 😄 ). The new website is still under development internally for now, but I believe it will be open source when we launch it. It's been on the back burner recently, but we hope to get it up this year.
